docs/implementation-references/getting-started-guide.md

```diff
@@ -54,23 +58,67 @@ sudo docker-compose up --build
-[ERROR] WARNING: An illegal reflective access operation has occurred
-[ERROR] WARNING: Illegal reflective access by org.apache.catalina.loader.WebappClassLoaderBase to field java.io.ObjectStreamClass$Caches.localDescs
-[ERROR] WARNING: Please consider reporting this to the maintainers of org.apache.catalina.loader.WebappClassLoaderBase
-[ERROR] WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
-[ERROR] WARNING: All illegal access operations will be denied in a future release
-```
+!!!WARNING
+
+    During the build stage, the following error messages may be displayed on the console. These error messages
+    are a result of the discrepancy of logging levels during the build stage and can be ignored.
+
+            [ERROR] Picked up _JAVA_OPTIONS: -Dhttp.proxyHost= -Dhttp.proxyPort= -Dhttps.proxyHost= -Dhttps.proxyPort=
+            [ERROR] WARNING: An illegal reflective access operation has occurred
+            [ERROR] WARNING: Illegal reflective access by org.apache.catalina.loader.WebappClassLoaderBase to field java.io.ObjectStreamClass$Caches.localDescs
+            [ERROR] WARNING: Please consider reporting this to the maintainers of org.apache.catalina.loader.WebappClassLoaderBase
+            [ERROR] WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
+            [ERROR] WARNING: All illegal access operations will be denied in a future release
+
+## Quick Overview of FDO
+
+FDO contains 3 major server-side components and 1 client-side component.
+The server-side components include Manufacturer, RV & Owner Service.
+The client-side includes device implementation in Java (PRI) or C (Client-sdk-fidoiot).
+
+FDO consists of four sets of protocols namely **DI, TO0, TO1 & TO2**.
+
+1. **DI (Device Initialization protocol)**
+
+    - Between Device & Manufacturer (**msg 10-13**).
+    - Initiated by device and device contacts Manufacturer Service.
+    - Includes creation & insertion of FDO credentials into newly manufactured device. Credential includes rvInfo which is used by device to connect with RV during T01 protocol
+    - Customers can take ownership of device after DI by extending the Ownership voucher to a particular customer.
+
+
+2. **TO0 (Transfer of Ownership 0 Protocol)**
+
+    - Between Owner & Rendezvous (RV) server (**msg 20-23**).
+    - Initiated by Owner once it receives Ownership voucher and Owner contacts RV server.
+    - TO0 creates a mapping of GUID with owner address and is stored in RV server's DB.
+    - Basically it creates a mapping like GUID=>OwnerAddress
+    - OwnerAddress can be DNS/IP or combination of both.
+
+
+3. **T01 (Transfer of Ownership 1 Protocol)**
+
+    - Between Device & Rendezvous (RV) server (**msg 30-33**).
+    - Initiated by Device and contacts RV server using the rvInfo directives collected DI.
```

Comment:
1. initiated by device . The device contacts
2. collected during DI

Reply:
Updated.